setup.py

```diff
@@ -19,7 +19,7 @@
         'astroid<2;python_version<"3"',
         'Jinja2',
         'PyYAML',
-        'sphinx>=1.6',
+        'sphinx>=1.6,<2',
```

Comment:
Was something broken with Sphinx 2 for you? The tests are passing with Sphinx 2 and I'm using it locally without issues.

Reply:
Yea, tests in travis were failing with them. I realize that probably shouldn't be in this PR, but I just wanted to see if this change was breaking tests. I can remove it 👍